Gemfile.lock

```diff
@@ -81,7 +81,7 @@ GEM
     daemons (1.2.2)
     descendants_tracker (0.0.4)
       thread_safe (~> 0.3, >= 0.3.1)
-    dotenv (2.0.1)
+    dotenv (0.7.0)
```

Comment:
Ah, I see. I can't see any major harm, given that we only really use Dotenv in dev anyway